web/template/layout/pagination.gotmpl

```diff
@@ -1,43 +1,72 @@
 {{ define "pagination" }}
-  <nav class="moj-pagination govuk-!-margin-bottom-1 govuk-!-margin-top-1 govuk-!-padding-right-2 govuk-!-padding-left-2 ignore-pagination-bottom" id="pagination-label">
-
-    <p class="govuk-visually-hidden" id="pagination-form-heading">Pagination navigation</p>
-
+<nav class="moj-pagination govuk-!-margin-bottom-1 govuk-!-margin-top-1 govuk-padding-right-2 govuk-!-padding-left-2 ignore-pagination-bottom" id="pagination-label"  role="navigation" aria-label="results">
     <div class="flex-container">
-
-      <p class="moj-pagination__results">Showing <b>
-      {{ if le .PageDetails.ShowingLowerLimit .TaskList.TotalTasks }}
-        {{ .PageDetails.ShowingLowerLimit }}
-      {{ else }}
-        {{.TaskList.TotalTasks}}
-      {{ end }}</b> to <b>
-      {{ if le .PageDetails.ShowingUpperLimit .TaskList.TotalTasks }}
-        {{ .PageDetails.ShowingUpperLimit }}
-      {{ else }}
-        {{.TaskList.TotalTasks}}
-      {{ end }}</b> of <b>{{ .TaskList.TotalTasks }}</b> tasks</p>
-
-      <ul class="moj-pagination__list">
-        <li class="moj-pagination__item moj-pagination__item--prev">
-          <button type="submit" a class="moj-pagination__link moj-pagination__link--btn" name="page" value="{{ .PageDetails.PreviousPage }}" {{ if lt .TaskList.Pages.PageTotal 2 }}disabled{{ end }} {{ if eq .TaskList.Pages.PageCurrent 1 }}disabled{{ end }}> Previous </button>
-        </li>
+        <p class="moj-pagination__results">Showing <b>
+        {{ if le .PageDetails.ShowingLowerLimit .TaskList.TotalTasks }}
+          {{ .PageDetails.ShowingLowerLimit }}
+        {{ else }}
+          {{.TaskList.TotalTasks}}
+        {{ end }}</b> to <b>
+        {{ if le .PageDetails.ShowingUpperLimit .TaskList.TotalTasks }}
+          {{ .PageDetails.ShowingUpperLimit }}
+        {{ else }}
+          {{.TaskList.TotalTasks}}
+        {{ end }}</b> of <b>{{ .TaskList.TotalTasks }}</b> tasks</p>
         {{$currentPage := .TaskList.Pages.PageCurrent}}
+        <div id="numbered-pages">
+          {{if ne $currentPage 1 }}
+              <div id="previous-page-pagination-link" class="govuk-pagination__prev">
+                <button type="submit" a class="govuk-link govuk-pagination__link moj-button" name="page" value="{{ .PageDetails.PreviousPage }}" rel="prev">
```

Comment:
Did you need to `a` just before the class?

Reply:
Yup because it's using an inbuilt a link class from GDS styling

Reply:
There's no need for the `a` it'll just confuse parsers as there's no `a` attribute on a button.